modules/luci-mod-info-dashboard/htdocs/luci-static/resources/view/info-dashboard/css/custom.css

```diff
@@ -0,0 +1 @@
tatus-container{text-align:center;margin-top:20px}.internet-icon{margin-bottom:10px}.internet-status{margin-top:10px;padding:10px;display:flex;margin:auto;flex-wrap:wrap}.internet-card{background:#f9f9f9;border:1px solid #ddd;border-radius:8px;padding:20px;width:50%;box-shadow:0 4px 8px rgba(0,0,0,.1);transition:transform .2s ease-in-out}.internet-card:hover{transform:scale(1.05)}.internet-card h3{display:flex;align-items:center;font-size:1.4rem;margin-bottom:15px;color:#333}.status-line{margin-bottom:10px;font-size:1rem}.status-label{font-weight:bold;color:#666}.status-value{font-weight:normal;color:#333}.status-value.connected{color:#28a745}.status-value.disconnected{color:#dc3545}.cbi-progressbar-infolink{border:1px solid var(--border-color-high);border-radius:5px;position:relative;min-width:170px;height:12px;background:#e0e0e0;margin:1.4em 0 4px;box-shadow:inset 0 2px 4px rgba(0,0,0,.3)}.cbi-progressbar-infolink>div{background:linear-gradient(to right,#3e55e4,#87d7ee);height:100%;transition:width .25s ease-in,box-shadow .25s ease-in;width:0;border-radius:5px;box-shadow:0 0 5px rgba(0,0,0,.5)}.cbi-progressbar-infolink::before{position:absolute;top:-1.4em;left:0;content:attr(title);white-space:pre;overflow:hidden;text-overflow:ellipsis;font-weight:bold;color:#333;padding:0 5px}.internet-card:hover .status-value{color:#000}@media (max-width:768px){.internet-card{width:100%}}.device-count{background:#f9f9f9;border:1px solid #ddd;border-radius:8px;padding:20px;box-shadow:0 4px 8px rgba(0,0,0,.1);transition:transform .2s ease-in-out;margin-bottom:20px;text-align:center}.connected-title{font-size:1.8em;font-weight:bold;color:black;margin-bottom:10px}.connected-count{font-size:2.5em;font-weight:bold;color:#007BFF;padding:10px 15px;border-radius:5px}.leases-title{font-size:1.5em;font-weight:bold;color:#333;margin:30px 0 10px}.table{width:100%;border-collapse:collapse}.table th,.table td{border:1px solid #ddd;padding:12px;text-align:left}.table th{background-color:black;color:white}.table tr:nth-child(even){background-color:#f9f9f9}.table tr:hover{background-color:#c4c4c4}.icon{position:relative}.notification-badge{position:absolute;top:0;right:0;background-color:rgb(255,230,0);color:black;border-radius:50%;padding:.5em;font-size:1.1em;font-weight:bold;line-height:.9;min-width:1.8em;text-align:center;display:block}.icon-container{display:flex;flex-direction:column;align-items:center;margin:20px;cursor:pointer}.icon-label{margin-top:1px;font-size:14px;font-weight:360;color:#000;text-align:center}.icon-container .icon-label.active{color:#45c8d4;font-weight:700}.internet-badge{width:24px;height:24px;background-size:contain;background-repeat:no-repeat;background-position:center;display:block;color:transparent}.dhcp-badge{display:block}.h2-System-Information,.h2-Information,.h2-Storage-Information{font-weight:700;color:black;font-size:24px;line-height:36px}.h2-System-Information{padding-bottom:.5em}.h2-Information,.h2-Storage-Information{padding:2em 0 .4em}
```

Comment:
Please don't do this. It shall be readable by reviewers. Let the compilers optimise it for install.